tests/generic_config_updater/test_ecn_config_update.py

```diff
@@ -97,7 +97,16 @@ def test_ecn_config_updates(duthost, ensure_dut_readiness, configdb_field, opera
     ecn_data = duthost.shell('sonic-db-cli CONFIG_DB hgetall "WRED_PROFILE|AZURE_LOSSLESS"')['stdout']
     ecn_data = ast.literal_eval(ecn_data)
     for field in configdb_field.split(','):
-        value = int(ecn_data[field]) + 1
+        value = int(ecn_data[field])
+        if "probability" in field:
+            if 0 <= value <= 99:
+                value += 1
+            elif value == 100:
+                value -= 1
```

Comment:
> FIx functionality issue on MRC skus in which the green_drop_probability can be 100.
> If the value equals to 100, it adds to it 1, and then the value 101 is failing on yang validation.

this PR feels a bit weird... I feel this should be a sonic-yang change, if 100 is not valid. and also the change is not restricting the hwsku as well.

also the description doesn't match with the code. the code says "if it is 100, then -1 make it 99 to be valid"